postgresql/add_unit_of_measure.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO unit_of_measure
```

Comment:
@prashantsable1991 The units of measure need to be added on openmrs not openElis